v3/src/components/common/edit-formula-modal.scss

```diff
@@ -1,10 +1,23 @@
 @use "../vars";
 
 .codap-modal-content {
+  min-height: 180px;
+  min-width: 400px;
```

Comment:
These values appear here and in the JS. They should be shared. See comment elsewhere for more details.